addons/l10n_uy/demo/account_demo.py

```diff
@@ -1,20 +1,523 @@
+        )
+        purchase_journal = self.env['account.journal'].search(
+            domain=[
+                *self.env['account.journal']._check_company_domain(cid),
+                ('type', '=', 'purchase')
+            ], limit=1,
+        )
+        return {
+            # Customer invoice demo
+            'demo_invoice_1': {
+                'company_id': company.id,
+                'move_type': 'out_invoice',
+                'partner_id': 'base.res_partner_4',
+                'journal_id': sale_journal.id,
+                'invoice_payment_term_id': 'account.account_payment_term_end_following_month',
+                'invoice_date': time.strftime('%Y-%m-01'),
+                'invoice_line_ids': [
+                    Command.create({'product_id': 'product.product_product_2', 'price_unit': 642.0, 'quantity': 1}),
+                    Command.create({'product_id': 'product.product_product_12', 'price_unit': 120.0, 'quantity': 4}),
+                    Command.create({'product_id': 'product.product_product_16', 'price_unit': 25.0, 'quantity': 20}),
+                    Command.create({'product_id': 'product.product_product_20', 'price_unit': 1950.0, 'quantity': 4}),
+                ],
+            },
+            'demo_invoice_2': {
+                'company_id': company.id,
+                'move_type': 'out_invoice',
+                'partner_id': 'base.res_partner_4',
+                'invoice_user_id': 'base.user_demo',
+                'journal_id': sale_journal.id,
+                'invoice_payment_term_id': 'account.account_payment_term_end_following_month',
+                'invoice_date': time.strftime('%Y-%m-05'),
+                'l10n_latam_document_type_id': 'l10n_uy.dc_e_inv',
+                'invoice_line_ids': [
+                    Command.create({'product_id': 'product.product_product_27', 'price_unit': 642.0, 'quantity': 5}),
+                    Command.create({'product_id': 'product.product_product_25', 'price_unit': 3245.0, 'quantity': 2}),
+                    Command.create({'product_id': 'product.consu_delivery_01', 'price_unit': 4000.0, 'quantity': 5}),
+                ],
+            },
+            'demo_invoice_3': {
+                'company_id': company.id,
+                'move_type': 'out_invoice',
+                'partner_id': 'l10n_uy.partner_cfu',
+                'invoice_user_id': 'base.user_demo',
+                'journal_id': sale_journal.id,
+                'invoice_payment_term_id': 'account.account_payment_term_end_following_month',
+                'invoice_date': time.strftime('%Y-%m-10'),
+                'l10n_latam_document_type_id': 'l10n_uy.dc_e_ticket',
+                'invoice_line_ids': [
+                    Command.create({'product_id': 'product.product_product_27', 'price_unit': 642.0, 'quantity': 5}),
+                    Command.create({'product_id': 'product.product_product_2', 'price_unit': 642.0, 'quantity': 1}),
+                    Command.create({'product_id':
```

Comment:
You still have the create on 2 lines for the products that begin with `'product.product_delivery`